bin/start_aws.sh

```diff
@@ -151,6 +151,12 @@ else
   pulumi config set aws:region -C "${script_dir}/../pulumi/python/config" "${AWS_DEFAULT_REGION}"
 fi
 
+# Function to auto-generate passwords
+function createpw() {
+  base64 /dev/random | tr -dc '[:alnum:]' | head -c${1:-16}
+  return 0
```

Comment:
Question:  this is may be my bash ignorance but how does the password get returned if we `return 0` at the end?